device-types/Ubiquiti/U6-LR.yaml

```diff
@@ -1,10 +1,21 @@
 ---
 manufacturer: Ubiquiti
-model: U6-LR
-slug: u6-lr
+model: Access Point U6 Long-Range
+part_number: U6-LR
+slug: ubiquiti-u6-lr
 u_height: 0
 is_full_depth: false
-comments: '[UniFi 6 Long Range Access Point](https://store.ui.com/products/unifi-6-long-range-access-point)'
+comments: |
+  [Access Point U6 Long-Range](https://store.ui.com/products/unifi-6-long-range-access-point)
+airflow: passive
+weight: 800.0
+weight_unit: g
 interfaces:
-  - name: lan0
+  - name: eth0
     type: 1000base-t
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: wlan0
+    type: ieee802.11n
```

Comment:
Where are you getting `n` from? This is an ax access point

Reply:
```suggestion
    type: ieee802.11x
```